components/Python/fileConnectors/file_to_dataframe/main.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import print_function
+
+import argparse
+import sys
+import time
+import os
+import pandas
+
+from parallelm.components import ConnectableComponent
+from parallelm.mlops.stats.multi_line_graph import MultiLineGraph
+from parallelm.mlops import mlops as mlops
+
+class MCenterComponentAdapter(ConnectableComponent):
+    """
+    Adapter for read_file_to_df 
+    """
+
+    def __init__(self, engine):
+        super(self.__class__, self).__init__(engine)
+
+    def _materialize(self, parent_data_objs, user_data):
+        file_path = str(parent_data_objs[0])
+        if file_path is None:
+            file_path = self._params.get('file_path')
+        return [read_file_to_df(file_path)]
+
+
+def read_file_to_df(filepath):
+    """
+    Read file and return DataFrame
+    """
+    mlops.init()
+    if not os.path.exists(filepath):
+        print("stderr- failed to find {}".format(filepath), file=sys.stderr)
+        raise Exception("file path does not exist: {}".format(filepath))
+
+    test_data = pandas.read_csv(filepath)
```

Comment:
so it is a csv read. Maybe:
1) change the name of the component to include that.
2) Expose csv loading options - like header, and so on